File: virtualteachergenaidemo.client/src/models/VoiceSettings.ts
export class VoiceSettings {
    voiceName: string;
    gender: string;
    language: string;

    constructor(voiceName: string, gender: string, language: string) {
        this.voiceName = voiceName;
        this.gender = gender;
        this.language = language;
    }

    get displayName(): string {
        return `${this.language} ${this.gender}`;
    }
}

File: VirtualTeacherGenAIDemo.Server/Models/Storage/VoiceSettings.cs
using DocumentFormat.OpenXml.Office2013.Excel;

namespace VirtualTeacherGenAIDemo.Server.Models.Storage
{
    public class VoiceSettings
    {
        public string VoiceName { get; set; } = string.Empty;
                
        public string Gender { get; set; } = string.Empty;

        public string Language { get; set; } = string.Empty;

    }
}
